apps/platform/src/app/Header.tsx

```diff
@@ -40,7 +40,9 @@ export default function Header() {
 				</nav>
 
 				<div className='ml-auto'>
-					<MuqaConnectButton  variant='green' />
+					<Link href="https://coda.io/form/Zazelenimo-Split-obrazac-za-prijedloge_dbtTs0gGIcq" target='_blank' className='rounded-xl bg-green text-base font-normal px-10 py-3 text-white hover:opacity-85'>
```

Comment:
Extract this url to a config and reference from there.
Both here and in the homepage.

Reply:
added